presto-main/src/main/java/com/facebook/presto/operator/MergeSortComparator.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator;
+
+import com.facebook.presto.spi.Page;
+
+public interface MergeSortComparator
+{
+    int compareTo(Page left, int leftPosition, Page right, int rightPosition);
```

Comment:
You could resist temptation to mimic `Comparator` and use more meaningful return type here (an enum)... (I didn't read other code, so it may not make any sense though)